### PR TITLE
FARM rewards

### PR DESCRIPTION
**Pull request by dapp-whisperer**

### CODE REVIEW

**Comment by stanislavkozlovski** [comment]
How can I re-create the digg-airdrop.json to verify it with the one that is to be committed here?

File: assistant/rewards/aws_utils.py [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 
-    s3_clientobj = s3.get_object(Bucket=upload_bucket, Key=upload_file_key)
+    #s3_clientobj = s3.get_object(Bucket=upload_bucket, Key=upload_file_key)
     # console.print(s3_clientobj)
-    s3_clientdata = s3_clientobj["Body"].read().decode("utf-8")
+    #s3_clientdata = s3_clientobj["Body"].read().decode("utf-8")
 
-    return s3_clientdata
+    #return s3_clientdata
+    url = "https://d5haax09x7ee2.cloudfront.net/rewards/{}".format(fileName)
```
Comment (stanislavkozlovski) [inline assistant/rewards/aws_utils.py:22]: What is this?

File: scripts/rewards/verify_rewards.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+console = Console()
+
+gas_strategy = GasNowStrategy("rapid")
+
+def test_main():
+    main()
+
+def main():
+    badger = connect_badger(badger_config.prod_json, load_keeper=True, load_guardian=True)
+    pendingContentHash = "0x07baafa95bf7c39ba948753dda64b2e82854b5a7597f6de9e15c23a96d0bfad3"
```
Comment (stanislavkozlovski) [inline scripts/rewards/verify_rewards.py:25]: How was this hash computed?

File: assistant/rewards/aws_utils.py (hunk 1 above)
Comment (Darruma) [reply assistant/rewards/aws_utils.py:22]: I wasn't able to get my AWS configuration setup properly so I am using this endpoint to load in the rewards file.

File: scripts/rewards/verify_rewards.py (hunk 2 above)
Comment (Darruma) [reply scripts/rewards/verify_rewards.py:25]: The hashes are computed from the merkle root of the pending  cycle . The current system works by encoding all the information about the claims in a [merkle tree](https://github.com/Darruma/badger-system/blob/master/assistant/rewards/merkle_tree.py). In this case the hash is hardcoded for testing purposes.

**Pull request merged**